src/components/header/header.js

```diff
@@ -64,6 +65,12 @@ const Header = props => (
           title: props.t('send'),
           childComponent: Send,
         })}>{props.t('send')}</Button>
+        <IconMenu
+          className={`${styles.iconButton} ${offlineStyle.disableWhenOffline}`}
+          icon='language' position='topRight'>
```

Comment:
The `IconMenu` can use `selectable` and `selected` properties to indicate selected language.
http://react-toolbox.com/#/components/menu